pkg/streamingpromql/operators/aggregations/count.go

```diff
@@ -87,7 +87,9 @@ func (g *CountGroupAggregationGroup) ComputeOutputSeries(_ types.ScalarData, tim
 		}
 	}
 
-	types.Float64SlicePool.Put(g.values, memoryConsumptionTracker)
-
 	return types.InstantVectorSeriesData{Floats: floatPoints}, false, nil
 }
+
+func (g *CountGroupAggregationGroup) Close(memoryConsumptionTracker *limiting.MemoryConsumptionTracker) {
+	types.Float64SlicePool.Put(g.values, memoryConsumptionTracker)
```

Comment:
This doesn't look safe to close multiple times. Granted this isn't an operator so the same expectation may not apply, but I wonder if we should be consistent?
(same for other changes in this pr)

Reply:
Done: a333aab